app/stac_api/serializers.py

```diff
@@ -615,6 +617,14 @@ class Meta:
     updated = serializers.DateTimeField(read_only=True)
 
     def validate(self, attrs):
```

Comment:
some docstring

Reply:
Well the name is quite explicit and it is a django method which is already documented by django, so I don't think docstring add any value here. It is and should only be called by django internals.